improver/calibration/__init__.py

```diff
@@ -209,6 +209,45 @@ def split_forecasts_and_coeffs(
     )
 
 
+def split_forecasts_and_bias_files(cubes: CubeList) -> Tuple[Cube, Optional[CubeList]]:
+    """Split the input forecast from the forecast error files used for bias-correction.
+
+    Args:
+        cubes:
+            A list of input cubes which will be split into forecast and forecast errors.
+
+    Returns:
+        - A cube containing the current forecast.
+        - If found, a cube or cubelist containing the bias correction files.
+
+    Raises:
+        ValueError: If multiple forecast cubes provided, when only one is expected.
+        ValueError: If no forecast is found.
+    """
+    forecast_cube = None
+    bias_cubes = CubeList()
+
+    for cube in cubes:
+        if "forecast_error" in cube.name():
+            bias_cubes.append(cube)
+        else:
+            if forecast_cube is None:
+                forecast_cube = cube
+            else:
+                msg = (
+                    "Multiple forecast inputs have been provided. Only one is expected."
+                )
+                raise ValueError(msg)
+
+    if forecast_cube is None:
+        msg = "No forecast is present. A forecast cube is required."
+        raise ValueError(msg)
+
+    bias_cubes = bias_cubes if bias_cubes else None
```

Comment:
Are there cases where a user would intentionally pass an empty list of bias cubes? Maybe this should trigger a warning?

Reply:
When setting up the CLI, the intention was that if no bias terms are passed in then the forecast should be returned. This is to enable to CLI to execute without error during spin-up. Maybe we should add a warning when this occurs so that users are aware of this behaviour in situations it is unintended?

Reply:
Ok, that makes sense, I didn't think of the spin-up case. Up to you whether you think it needs a warning.

Reply:
I've added a warning and associated behaviour in line with how `apply_emos_coefficients` CLI handles this situation. I've updated improver_test_data to add a new kgo for this new case (see PR: https://github.com/metoppv/improver_test_data/pull/47).